workspace/src/main/kotlin/com/cosmotech/workspace/service/WorkspaceServiceImpl.kt

```diff
@@ -195,6 +204,14 @@ internal class WorkspaceServiceImpl(
     if (workspace.security != null && existingWorkspace.security == null) {
```

Comment:
remove and prevent modification of security. security modification should be done in endpoint security